test/pylib/manager_client.py

```diff
@@ -180,15 +180,18 @@ async def replicas(self) -> int:
         """Get number of configured replicas for the cluster (replication factor)"""
         return await self.client.get_json("/cluster/replicas")
 
-    async def running_servers(self) -> list[ServerInfo]:
-        """Get List of server info (id and IP address) of running servers"""
+    async def running_servers(self, *server_ids: int) -> list[ServerInfo]:
```

Comment:
The caller of this function can intersect the returned list with their chosen filter. No need to complicate the function itself.